web/client/selectors/geostory.js

```diff
@@ -72,6 +72,11 @@ export const canEditSelector = state => get(resourceSelector(state), 'canEdit',
  * @param {object} state the application state
  */
 export const geostoryIdSelector = state => get(resourceSelector(state), 'id');
+/**
+ * Get name/title of current story
+ * @param {object} state the application state
```

Comment:
is this an object or a string?

Reply:
state is object and return is string, I have updated with return